troubleshooting-order/solutions/potential-customers/src/main/resources/application.properties

```
@@ -0,0 +1,2 @@
+# Streams configuration
+quarkus.kafka-streams.topics = potential-customers-detected,potential-customers-average
```

Remove this unused topic, otherwise the application won't start

```suggestion
quarkus.kafka-streams.topics = potential-customers-detected
```